src/trace-api.js

```diff
@@ -177,6 +178,22 @@ TraceAgent.prototype.runInRootSpan = function(options, fn) {
 };
 
 /**
+ * Returns a unique identifier for the current root span. This can be used to
+ * uniquely identify the current root span. If there is no current root span,
+ * or if we have lost context, this will return null. The structure and the
+ * length of the returned string should be treated opaquely - the only guarantee
+ * is that the value would unique for every root span.
+ * @returns {string} the root span id, or null if it doesn't exist
+ */
+TraceAgent.prototype.getRootSpanId = function() {
```

Comment:
Yeah, I think I'm OK with calling it `getRootSpanId` given the context of the JS API.

Eventually if we separate out the context propagation library from the trace library and make it a shared dependency of logging and trace, then we could re-evaluate what to name it then.